awslimitchecker/runner.py

```diff
@@ -358,6 +361,11 @@ def console_entry_point(self):
             ta_refresh_timeout=args.ta_refresh_timeout
         )
 
+        if args.skip_service is not None:
```

Comment:
This conditional is not needed at all; the ``add_argument`` call for ``--skip-service`` can have ``default=[]`` added to it to produce an empty list if not specified; that can be passed directly in to ``remove_skipped_services``.